defaults/main.yml

```diff
@@ -33,3 +33,5 @@ nomad_client: false
 nomad_client_node_class: worker
 nomad_client_host_volumes: []
 nomad_client_host_networks: []
+nomad_docker_auth_enabled: false
+nomad_docker_auth_file: /root/.docker/config.json
```

Comment:
This var is not asserted anymore

Reply:
`- name: Test if nomad_docker_auth_config is set correctly
  ansible.builtin.assert:
    that:
      - nomad_docker_auth_config is defined
  when:
    - nomad_docker_auth_enabled | bool`
    
    and nomad_docker_auth_enabled defaults to `false`

Reply:
`nomad_docker_auth_file` != `nomad_docker_auth_config`